### PR TITLE
Adding doublespace check

### PR DESCRIPTION
**Pull request by y0nd3rb0y**
Resolving issue #2787 

Before submitting pull request, please ensure that:

- [X] Every commit has message which describes it
- [X] Every commit is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [X] Any code changes come with testcase
- [ ] Any new functionality is covered by user documentation

### CODE REVIEW

File: weblate/checks/chars.py [hunk 1]
```diff
@@ -137,6 +139,28 @@ def get_fixup(self, unit):
...
+
+    def check_single(self, source, target, unit):
+        # One letter things are usually decimal/thousand separators
+        if len(source) <= 1 and len(target) <= 1:
+            return False
+        if not source or not target:
+            return False
+
+        # Check if target contains double space
+        return bool(re.search(' {2,}', target))
```
Comment (nijel) [inline weblate/checks/chars.py]:
```suggestion
        return '  ' in target
```

I think this will perform way better than doing regexp search.

**Comment by y0nd3rb0y** [comment]
Thanks Michal @nijel . I've added your suggestions accordingly
Please let me know if I can do anything else for the check

**Comment by nijel** [comment]
Thanks. The code looks good now, the only missing piece is documentation in https://github.com/WeblateOrg/weblate/blob/master/docs/user/checks.rst

**Comment by nijel** [comment]
Thanks, I think it's not ready to be merged after bugfix 3.9.1 is released. It would be great if you could meanwhile squash all commits into one on top of current master (if not I will do it when merging).

**Pull request merged**